### ISSUE TITLE
Workshop planning & running

### ISSUE THREAD

**Issue by Journerdism** [18F/atf-eregs#269]

**Comment by jehlers** [comment]
Postponed to next sprint due to the snow storm. The workshop is all planned, but it is now going to happen on Feb 1.

**Comment by jehlers** [comment]
This workshop went off without a hitch on February 1st! (No snow issues! :smile: :snowman: :tada: )

Here are some takeaways and what came out of the workshop:

# 2/1 eRegs Workshop with ATF
#### **Main question:** How do we customize eRegs to meet ATF’s needs and regulatory environment?

## Exercise 1 - Design studio

**Set up:** We split up into two groups, half in person and half remote via google hangouts. Participants were given 7 minutes to sketch as many ideas as they could on small sheets of paper. One idea per sheet of paper.

**Premise:** Based on the user research findings, what new features could help solve some of the problems? Participants were encouraged to think broadly, but if it was easier could focus on specific types of documents such as rulings, open letters, Q&As, etc… that could be brought into the tool.

**Results:**

General themes that were seen across almost all sketches:

- Cross linking / referencing across all document types is needed 
- A way to view documents other than regulations
- Good search that can search all document types in the tool
- Curated content
  - This can be either more plain language materials to complement the legal language;
  - Or, curated metadata that helps sort documents or information by topic to help make search or filtering better
- An intuitive structural relationship between documents is important to show
- Understanding that there are 2 types of users and paths to get to the information they want:
  - _Black diamonds_ - Power users who know the types of documents and information they are looking for. They will go straight to the regs, and need to be able to find other information that is pertinent to their current place on the site.
  - _Bunny slopes_ - New users, or users who do not interact with ATF daily, need to be guided to the information they need. These users will probably search first or benefit from more plain language documents first because they don’t know what they are looking for or how to decipher a regulation.

## Exercise 2 - Hierarchy exercise

**Set up:** The in person group was given 7 notecards with types of documents listed on them. The  group added 4 more notecards to the pile before starting. 

**Goal:** Organize the 11 notecards/ATF documents in order of importance. 
_The group came to the conclusion that “importance” meant “educational importance” and not necessarily legal importance or how often users visit a certain webpage._

**Round 1**
1. Q&As 
2. Regulations (CFR)
3. Forms
4. Rulings
5. Procedures
6. Publications
7. Newsletters
8. Open Letters
9. General Information Section (in the reg reference guide)
10. Industry Circular
11. Statutes (USC)

**Round 2 - Take 4 away.**
1. Q&As 
2. Regulations (CFR)
3. Forms
4. Rulings
5. Publications
6. Newsletters
7. Statutes (USC)

**Round 3 - Take 3 more away.**
1. Q&As 
2. Regulations (CFR)
3. Forms
4. Rulings

**Round 4 - Take 1 more away.**
1. Q&As 
2. Regulations (CFR)
3. Rulings

**Round 5 - Take 1 more away.**
1. Q&As 
2. Regulations (CFR)

**End result:** Q&As and regulations are valued more highly by the group than rulings or any other documents. 

After a lot of discussion Q&As were placed at the top of every list because they help to explain regulations in an easy to understand way. They are also the number 1 page, based on analytics, that users visit on atf.gov.

Regulations are already in the eRegulations tool. As someone said during the workshop, “Statutes (USC) are the foundation of ATF,” they are why ATF exists, but regulations are the bread and butter. They are what users need to be educated on.

This list could be the beginnings of a product roadmap. It allowed the group to agree on all of the research docs that could be put into the tool, and ordered them in a way that they could be built one-by-one. The ones on the top of the list are more known because they surfaced themselves in recent user interviews, those tools could be worked on while more research is done on the documents later in the list.

## Exercise 3 - Goal exercise

**Set up:** Everyone wrote 3 (or more) ideas or principles for this new feature on post-it notes. One idea per post-it. Then as a group, we organized them and named each group.

**_Results_**

**Goal statement:** ATF eRegulations should help visitors find the guidance they need with ease and confidence, by searching and navigating a logically-presented one-stop-shop for compliance education.

**Main features needed to accomplish this goal:**

- Great search
  - Based on folksonomy (user created terms), not just taxonomy (official terms)
  - Could include some agency curation to help with the above bullet
- Cross-referencing of all documents
  - Users should be able to see other documents that relate to the document they are currently reading even if the current document (like the regs) do not reference the other documents
  - User should be able to easily get to documents that are referenced in other documents
  - Could include some agency curation to help with the above bullets
- Logical organization/presentation of document relationships
  - These documents should be organized and presented in an intuitive way based on users’ current mental models
  - This organization can also help teach users how these documents relate to each other in the ATF arena
